HdcpProfile/HdcpProfile.cpp

```diff
@@ -167,6 +177,9 @@ namespace WPEFramework
             hdcpStatus["isConnected"] = isConnected;
             hdcpStatus["isHDCPCompliant"] = isHDCPCompliant;
             hdcpStatus["isHDCPEnabled"] = isHDCPEnabled;
+            if (1 < m_apiVersionNumber) {
```

Comment:
Pls help add a getter() in the base AbstractPlugin class to return the current version (m_currVersion). That can be used to read the version instead of m_apiVersionNumber